src/Appwrite/Utopia/Database/Validator/Queries/Users/ListUsers.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Appwrite\Utopia\Database\Validator\Queries\Users;
+
+use Appwrite\Utopia\Database\Validator\Queries\Collection;
+
+class ListUsers extends Collection
```

Comment:
I think this file can be under Validators/Queries and can be called `Users`

Reply:
How about later when we need multiple Validators per service?

Reply:
Answered below.

Reply:
We now have these validation classes. Resolving discussion.